src/screens/WhotoFollow/index.js

```diff
@@ -277,7 +282,7 @@ const WhotoFollow = () => {
           {'Find interesting people to follow.\nYou can edit this anytime.'}
         </Text>
       </View>
-      {isLoading ? <ActivityIndicator size="small" color="#0000ff" /> : null}
+      {isLoading ? <ActivityIndicator size="small" color={colors.signed_primary} /> : null}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Good job on replacing the hardcoded color value with a value from the `colors` utility module. This improves maintainability and consistency in your codebase.


<!-- This is an auto-generated comment by OSS CodeRabbit -->